README.md

```diff
@@ -413,7 +428,7 @@ ua version
 # Make apt aware of the ESM source files
 sudo apt update
 # Generates ubuntu-advantage-tools messages that should be delivered to MOTD
-# This script is triggered by a systemd timer twice a day. To test it, we need
+# This script is triggered by the systemd timer 4 times a day. To test it, we need
 # to enforce that it was already executed.
 sudo systemctl start ua-timer.service
 # Force updating MOTD messages related to update-notifier
```

Comment:
As this is not in `lib` anymore, should it be updated/removed? How do we call this now?